README.md

```diff
@@ -37,8 +37,29 @@ User: A person that uses our software.
 
 Cantis also generates its own [glossary](glossary.txt).
 
+## Install
+In order to use the `@GlossaryTerm` annotation, you must add a dependency to `cantis` to your project:
```

Comment:
This dependency is not strictly necessary. The readme already states:
> The annotation @GlossaryTerm can be placed on the classpath in many ways,
for example via the maven dependency `com.github.korthout:cantis`, but you can 
also define your own @GlossaryTerm annotation as long as it is called
@GlossaryTerm.

But maybe that's not clear, so perhaps we should change the note. I like the idea of an `Install` section, so maybe the note belongs here anyways.

Reply:
I don't think it's clear no. Your using `RetentionPolicy.CLASS` (by not specifying a `Retention`). This means that the scope must be compile, instead of provided. So there's a little more to it than simply "adding it to the classpath" IMHO.

Reply:
You're right. Just adding it to the classpath is not enough. But, you can declare your own @GlossaryTerm annotation and annotate your domain classes with it. Do you have a suggestion to improve the note?

Reply:
@nicojs I've move and change the note to be more informative. Do you agree with my changes?